contracts/chain-adapters/ZkStack_CustomGasToken_Adapter.sol

```diff
@@ -111,6 +125,18 @@ contract ZkStack_CustomGasToken_Adapter is AdapterInterface {
         MAX_TX_GASPRICE = _maxTxGasprice;
         L1_GAS_TO_L2_GAS_PER_PUB_DATA_LIMIT = _l1GasToL2GasPerPubDataLimit;
         SHARED_BRIDGE = BRIDGE_HUB.sharedBridge();
+        address zero = address(0);
+        if (address(_l1Usdc) != zero) {
+            if (_usdcSharedBridge == zero && address(_cctpTokenMessenger) == zero) {
+                revert InvalidBridgeConfig();
+            }
+
+            // Bridged and Native USDC are mutually exclusive.
+            if (_usdcSharedBridge != zero && address(_cctpTokenMessenger) != zero) {
+                revert InvalidBridgeConfig();
+            }
```

Comment:
Same readability nit as above. I think these checks are hard to understand interpret the way they are written here.